lib/strings.nix

```
@@ -269,6 +269,37 @@ rec {
     f:
     list: concatStringsSep sep (lib.imap1 f list);
 
+  /**
+    Like [`concatMapStringsSep`](#function-library-lib.strings.concatMapStringsSep)
+    but takes an attribute set instead of a list.
+
+    # Type
+
+    ```
+    concatMapStringAttrsSep :: String -> (String: Any: String) -> AttrSet -> String
+    ```
+
+    # Examples
+
+    :::{.example}
+    ## `lib.strings.concatMapStringAttrsSep` usage example
+
+    ```nix
+    concatMapStringAttrsSep "\n" (name: value: "${name}: foo-${value}") { a = "0.1.0"; b = "0.2.0"; }
+    => "a: foo-0.1.0\nb: foo-0.2.0"
+    ```
+
+    :::
+  */
+  concatMapStringAttrsSep =
```

Shouldn't this be

```suggestion
  concatMapAttrsStringSep =
```

?

---

Cool!